lib/acts_as_taggable_on/taggable/core.rb

```diff
@@ -28,12 +28,16 @@ def initialize_acts_as_taggable_on_core
             has_many context_taggings, -> { includes(:tag).order(taggings_order).where(context: tags_type) },
                      as: :taggable,
                      class_name: 'ActsAsTaggableOn::Tagging',
-                     dependent: :destroy
+                     dependent: :destroy,
+                     after_add: :dirty_tag_list,
```

Comment:
Shall we name the callback `dirtify_tag_list` instead? Same for the `after_remove` callback.